src/config/governence.json

```diff
@@ -6,18 +6,44 @@
   },
   "permissive": {
     "connections": {
-      "invitation": "always",
-      "request": "always"
+      "create-invitation": "always",
+      "receive-invitation": "always",
+      "accept-invitation": "always",
+      "accept-request": "always",
+      "send-ping": "always",
+      "remove": "always"
+    },
+    "proof": {
+      "proposal_sent": "always",
+      "proposal_received": "always",
+      "offer_sent": "always",
+      "offer_received": "always",
+      "request_sent": "always",
+      "request_received": "always",
+      "credential_issued": "always",
+      "credential_received": "always",
+      "credential_acked": "always"
     }
   },
   "SierraLeone": {
```

Comment:
I'm wondering if we should rename this something more generic, like SingleConnection - I'd like to keep any implementation specific wording out of our public repos.